### DISCUSSION TITLE
Openstack SDK: how to interface ?

### DISCUSSION THREAD

**Discussion by aglossa** [deepnoodle-ai/risor#85] [General]
Hello,

I am discovering risor and would evaluate it if Openstack API is easily accessible.

Does a direct integration similar to AWS or Postgres seems possible, for exemple with Gophercloud SDK https://github.com/gophercloud/gophercloud ?
Or do I need to use a more general Risor / Go interfacing ?

Best regards.

**Comment by myzie** [comment]
Hi @aglossa, integrating the Gophercloud should be possible, in the same way as the AWS and Postgres modules.

It might be worth chatting on Slack or Discord. Are you on gophers.slack.com by chance?

**Comment by aglossa** [comment]
In general, Europe has not fallen for Slack mania :-)
And I worked for a government...
